File: Redes Fup 2025.1/Semana 9/media.py
n = int(input())

n1 = []
n2 = []
media = []

i=0
while i<n:
    x = float(input())
    n1.append(x)
    i+=1

i=0
while i<n:
    x = float(input())
    n2.append(x)
    
    media.append((n1[i]+n2[i])/2)
    i+=1

i=0
while i<n:
    print(f"{media[i]:.2f} ")
    i+=1

File: SI Fup 2026.1/Aula04/media.c
#include <stdio.h>

int main()
{
	double prova1, prova2, trabalho;
	scanf("%lf", &prova1);
	scanf("%lf", &prova2);
	scanf("%lf", &trabalho);
	double media = (5 * prova1 + 3 * prova2 + 2 * trabalho) / 10.0;
	printf("%f\n", media);
}
